src/NuProj.Packages/NuProj.nuproj

```diff
@@ -51,7 +51,10 @@ There is also a Visual Studio extension which you find under http://bit.ly/NuPro
     <Content Include="$(BasePath)raw\Additional\Microsoft.Common.NuProj.targets">
       <Link>tools\Microsoft.Common.NuProj.targets</Link>
     </Content>
-    <Content Include="..\packages\NuGet.CommandLine.2.8.6\tools\NuGet.exe">
+    <Content Include="..\NuGet.VSTS\tools\CredentialProvider.VSS.exe">
```

Comment:
Are these binaries available via NuGet so we don't have to check in binaries to git?